website/project/utils.py

```diff
@@ -118,3 +118,41 @@ def sizeof_fmt(num, suffix='B'):
             return '%3.1f%s%s' % (num, unit, suffix)
         num /= 1000.0
     return '%.1f%s%s' % (num, 'Y', suffix)
+
+
+def get_storage_limits_css(node):
+    status = node.storage_limit_status
+    public_limit = node.custom_storage_usage_limit_public or settings.STORAGE_LIMIT_PUBLIC
+    private_limit = node.custom_storage_usage_limit_private or settings.STORAGE_LIMIT_PRIVATE
+
+    over_public_size = sizeof_fmt(int(public_limit * settings.GBs))
+    over_private_size = sizeof_fmt(int(private_limit * settings.GBs))
+
+    if status is settings.StorageLimits.APPROACHING_PRIVATE:
+        return {
+            'class': 'btn-warning storage-warning',
+            'text': f'This project is approaching the storage limit of {over_private_size}. To learn more about limits and alternative storage options visit https://help.osf.io/.',
```

Comment:
According to the [requirements](https://docs.google.com/document/d/13dZBXO_JqZOcf4fe-HKCk_YDMA5PYokfOOw4KB3jbEY/edit?ts=5f47d7b8#bookmark=id.ga5sxbogg83u), this should be "This project/component is approaching the storage limit for OSF Storage. To learn more about limits and alternative storage options visit our help docs."  And “visit our help docs” is a stable link to a help doc with options and instructions. You should be able to get that link from Kayla. The rest of these should be similarly updated.

Or can we not do a link in the help text?

Reply:
We can't do a link like that in tooltip text, it's not clickable.